### PR TITLE
Bug fix in the fusion of the different orientations

### PR DESCRIPTION
**Pull request by gurayerus**

### CODE REVIEW

File: DeepMRSeg/deepmrseg_apply.py [hunk 1]
```diff
@@ -43,36 +39,30 @@ def read_flags(argv):
...
 	parser = _argparse.ArgumentParser( formatter_class=_argparse.RawDescriptionHelpFormatter, \
 		description=descTxt, epilog=epilogTxt )
 
-#	TASK
+#	TASK/MODEL
 #	===========
-	dirArgs = parser.add_argument_group( 'TASK')
-	dirArgs.add_argument( "--task", default=None, type=str, \
+	dirArgs = parser.add_argument_group( 'MODEL')
+	dirArgs.add_argument( "--model", default=None, type=str, \
```
Comment (ashishsingh18) [inline DeepMRSeg/deepmrseg_apply.py:65]: Why did we remove the '--task' parameter? That was much more intuitive.

File: README.md [hunk 2]
```diff
@@ -84,33 +84,26 @@ We provide here few examples using minimal argument sets as a quick reference. T
...
-#   WM lesion segmentation model requires both FL and T1 scans 
+# Segment single subject (multi-modality task)
 #   Img names for different modalities are entered as repeated args in the same order
 #   used in model training
-deepmrseg_apply --task wmlesion --inImg subj1_FL.nii.gz --inImg subj1_T1.nii.gz --outImg subj1_T1_WMLES.nii.gz     
-
-# Batch processing of multiple subjects using a subject list
-#   User provides a csv file with columns: ID,InputT1,OutputImage
-deepmrseg_apply --task brainmask --sList subjectList.csv
+deepmrseg_apply --model wmlesion --inImg subj1_FL.nii.gz --inImg subj1_T1.nii.gz --outImg subj1_T1_WMLES.nii.gz
```
Comment (ashishsingh18) [inline README.md:97]: This line could be removed for now. We can add it when we have the model for 'wmlesion'.

File: DeepMRSeg/deepmrseg_apply.py [hunk 3]
```diff
@@ -43,36 +39,30 @@ def read_flags(argv):
 
 	epilogTxt = '''Examples:
 	
-  ## Apply bmask model on single image (I/O OPTION 1)
-  deepmrseg_downloadmodel --model bmask
-  {prog} --task bmask --inImg sub1_T1.nii.gz --outImg sub1_bmaskseg.nii.gz
+  ## Apply dlicv model on single image (I/O OPTION 1)
+  deepmrseg_downloadmodel --model dlicv
+  {prog} --model dlicv --inImg sub1_T1.nii.gz --outImg sub1_DLICV.nii.gz
```
Comment (ashishsingh18) [inline DeepMRSeg/deepmrseg_apply.py:44]: _{prog}_ should be replaced with actual name so users can just copy paste and don't have to think.

**Review by ashishsingh18 — CHANGES_REQUESTED** [review]
Please see some comments.

File: DeepMRSeg/deepmrseg_apply.py (hunk 1 above)
Comment (gurayerus) [reply DeepMRSeg/deepmrseg_apply.py:65]: I also liked --task more, but changed it to --model to make it consistent with downloadmodel script, e.g.:
  deepmrseg_downloadmodel --model muse
  deepmrseg_apply --model muse

File: README.md (hunk 2 above)
Comment (gurayerus) [reply README.md:97]: I will do

File: DeepMRSeg/deepmrseg_apply.py (hunk 3 above)
Comment (gurayerus) [reply DeepMRSeg/deepmrseg_apply.py:44]: I agree

Comment (gurayerus) [reply DeepMRSeg/deepmrseg_apply.py:44]: I checked this. prog here is a variable and it's replaced with the program name, so it's good.

File: DeepMRSeg/deepmrseg_apply.py (hunk 1 above)
Comment (gurayerus) [reply DeepMRSeg/deepmrseg_apply.py:65]: I convert it back to task , following your suggestion

**Comment by ashishsingh18** [comment]
@gurayerus Can you please take a look at the codacy issue too?

**Review by ashishsingh18 — APPROVED** [review]
looks fine.

**Pull request merged**